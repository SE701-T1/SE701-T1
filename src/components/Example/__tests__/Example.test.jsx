import React from 'react';
import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';

import Example from '../Example';

describe('Example Components', () => {
  it('should display example test', () => {
    render(<Example />);

    expect(screen.getByText('This is an example component.')).toBeInTheDocument();
  });

  // snapshot test
  it('produces correct snapshot of the example', () => {
    const tree = renderer.create(<Example />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
