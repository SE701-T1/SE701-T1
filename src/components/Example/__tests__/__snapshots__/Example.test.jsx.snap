// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Example Components produces correct snapshot of the example 1`] = `
<p
  className="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
>
  This is an example component.
</p>
`;
